utils/http.go

```diff
@@ -82,6 +86,22 @@ func InsecureSkipVerifyTLSConfig() *tls.Config {
 	return config
 }
 
+// TlsConfigCACert returns a new tls.Config with caCertFile added to the set of trusted CAs
+func TlsConfigCACert(caCertFile string) (*tls.Config, error) {
+	config := DefaultTLSConfig()
+	if pemBytes, err := ioutil.ReadFile(caCertFile); err == nil {
+		rpool, _ := x509.SystemCertPool()
+		if rpool.AppendCertsFromPEM(pemBytes) {
+			config.RootCAs = rpool
+		} else {
+			return nil, fmt.Errorf("Error appending CA certificate '%s' to pool", caCertFile)
```

Comment:
This can add more than one certificate - so better use plural in error message - here and.below.